src/modules/bppp/tests_impl.h

```diff
@@ -254,10 +254,14 @@ void test_serialize_two_points(void) {
     {
         secp256k1_ge X_tmp, R_tmp;
         unsigned char buf[65];
+        unsigned char invalid_sign;
         random_group_element_test(&X);
         random_group_element_test(&R);
         secp256k1_bppp_serialize_points(buf, &X, &R);
-        buf[0] |= 4 + (unsigned char)secp256k1_testrandi64(4, 255);
+        do {
+            invalid_sign = (unsigned char)secp256k1_testrand256();
+        } while(invalid_sign == 0x02 || invalid_sign == 0x03);
+        buf[0] = invalid_sign;
```

Comment:
why not just `buf[0] = 4 + (unsigned char)secp256k1_testrandi64(0, 253);`?

Reply:
Okay yes, this has the same effect...

Reply:
done